package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "webpack",
   "version": "1.0.0",
-  "description": "",
+  "description": "<a name=\"readme-top\"></a>",
   "private": true,
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
- [ ] I suggest that scripts should contain commands for easy and hitchfree running of the app, including building, commands like

```
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "watch": "webpack --watch",
    "start": "webpack serve --open",
    "build": "webpack"
  },
```

So that, when the app is to be ran, just simple `npm start` will start the app.